lib.php

```diff
@@ -66,3 +66,21 @@ function auth_saml2_after_require_login() {
 function auth_saml2_before_http_headers() {
     \auth_saml2\auto_login::process();
 }
+
+/**
+ * Add service status checks
+ *
+ * @return array of check objects
+ */
+function auth_saml2_status_checks() : array {
+    global $saml2auth;
+    require_once(__DIR__ . '/setup.php');
+
+    // If saml is not configured then return no check.
```

Comment:
Perhaps it is better if the comment is be updated to reflect the immediate logic.

"If configured, it will return any status checks for the saml service" or similar.